project3/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<lang="en">
```

Comment:
Missing opening `html` element:
```suggestion
<html lang="en">
```